packages/fxa-auth-server/test/local/payments/subscription-reminders.js

```diff
@@ -50,6 +50,7 @@ describe('SubscriptionReminders', () => {
   let mockStripeHelper;
   let reminder;
   let mockConfig;
+  let realDateNow;
```

Comment:
The changes for this has nothing to do with the functionality related to metadata/product config.  There was an issue with `Date.now` not being restored after a test failure, leading to an OOM error elsewhere.